manifests/service.pp

```diff
@@ -1,23 +1,37 @@
 # configure, enable, and start pulpcore services
 # @api private
 class pulpcore::service {
+  ['gunicorn', 'rq'].each |$bin| {
+    file { "${pulpcore::libexecdir}/${bin}":
```

Comment:
Let's package that instead: https://github.com/theforeman/pulpcore-packaging/pull/37

Reply:
https://github.com/theforeman/pulpcore-packaging/pull/41 is the cherry pick to 3.6.